media/css/firefox/mobile.scss

```diff
@@ -8,6 +8,7 @@
 @import '../hubs/masthead';
 @import '../hubs/sections';
 @import '../hubs/sub-nav';
+@import '../hubs/nav-download-button-overrides';
```

Comment:
Since this file removes the button and is not a good place to put other overrides can we name it something like `nav-download-button-remove`?